templates/collectd_metrics/template.yml

```diff
@@ -0,0 +1,11 @@
+skeleton_path: ../skeleton.json
+
+elasticsearch_template:
+  name: com.redhat.viaq-collectd
```

Comment:
We prefer to have the template with upstream brand as ovirt.viaq-collectd.
Can be org.ovirt.viaq-collectd.